global.json

```diff
@@ -5,12 +5,12 @@
       "dotnet": [
         "$(MicrosoftNETCoreApp60Version)",
         "$(MicrosoftNETCoreApp70Version)",
-        "$(VSRedistCommonNetCoreSharedFrameworkx6480Version)"
+        "$(MicrosoftNETCoreApp80Version)"
```

Comment:
Are we removing `VSRedistCommonNetCoreSharedFrameworkx6480Version` completely? Should we remove the declaration at [eng/Versions.props](https://github.com/dotnet/diagnostics/blob/de3b9329cf5d976d9e577e4002908426917c72d1/eng/Versions.props#L21)

Reply:
It will get removed when switch the runtime DARC updates to the 9.0 channel.  It is currently updated in eng/Versions.props by DARC.